hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/OMRequestTestUtils.java

```diff
@@ -447,6 +447,7 @@ public static OmKeyInfo createOmKeyInfo(String volumeName, String bucketName,
         .setVolumeName(volumeName)
         .setBucketName(bucketName)
         .setKeyName(keyName)
+        .setFileName(keyName)
```

Comment:
Is it typo or `fileName` and `keyName` are actually same?

Reply:
Good catch. In the test cases they are the same. But in prod it will extract the last element (just the file name):

https://github.com/apache/ozone/blob/3e83eb6800a269eafb6a77a63303a46ba5882e5c/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java#L698-L699

I will make the test helper more robust by wrapping it in `OzoneFSUtils.getFileName()` just like the prod one.

Reply:
Got it.